client/manager_sql.go

```diff
@@ -1,18 +1,31 @@
 package client
 
 import (
-	"encoding/json"
+	"fmt"
 	"github.com/imdario/mergo"
 	"github.com/jmoiron/sqlx"
 	"github.com/ory-am/fosite"
 	"github.com/pkg/errors"
+	"strings"
 )
 
 var sqlSchema = []string{
 	`CREATE TABLE IF NOT EXISTS hydra_client (
-	id      varchar(255) NOT NULL PRIMARY KEY,
-	version int NOT NULL DEFAULT 0,
-	client  json NOT NULL)`,
+	id      	varchar(255) NOT NULL PRIMARY KEY,
+	client_name  	text NOT NULL,
```

Comment:
I didn't know that, do you have any resources on that? In general it's nicer to deal with non-nullable strings in Go because Go doesn't know nil strings. tl;dr SQL null fields are a bit tricky to deal with in Go iirc.